packages/wonder-blocks-button/components/button-core.js

```diff
@@ -125,6 +125,9 @@ const sharedStyles = StyleSheet.create({
         outline: "none",
         textDecoration: "none",
         boxSizing: "border-box",
+        "::-moz-focus-inner": {
```

Comment:
Would be good to add a brief comment explaining what this does and why we added it (so that we don't accidentally remove it later!)